selfdrive/controls/controlsd.py

```diff
@@ -608,9 +611,10 @@ def state_control(self, CS):
                                                                                        lat_plan.psis,
                                                                                        lat_plan.curvatures,
                                                                                        lat_plan.curvatureRates)
-      actuators.steer, actuators.steeringAngleDeg, lac_log = self.LaC.update(CC.latActive, CS, self.VM, lp,
-                                                                             self.last_actuators, self.steer_limited, self.desired_curvature,
-                                                                             self.desired_curvature_rate, self.sm['liveLocationKalman'])
+      actuators.steer, actuators.steeringAngleDeg, actuators.curvature, lac_log = \
+        self.LaC.update(CC.latActive, CS, self.VM, lp,
+                        self.last_actuators, self.steer_limited, self.desired_curvature,
+                        self.desired_curvature_rate, self.sm['liveLocationKalman'])
```

Comment:
```suggestion
      actuators.steer, actuators.steeringAngleDeg, actuators.curvature, lac_log = \
        self.LaC.update(CC.latActive, CS, self.VM, lp,
                        self.last_actuators, self.steer_limited, self.desired_curvature,
                        self.desired_curvature_rate, self.sm['liveLocationKalman'])
```

We might want to start passing in `actuators` and the lateral controller can modify it